### PR TITLE
Uppercase variable names. iModel/project ids instead of names

### PR DESCRIPTION
**Pull request by williamkbentley**

### CODE REVIEW

File: src/frontend/app/AppState.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ export enum SwitchState {
 }
 
 export interface SelectedIModel {
-  imodelName: string;
-  projectName: string;
+  imodelId: string;
+  projectId: string;
```
Comment (calebmshafer) [inline src/frontend/app/AppState.ts:22]: Can we switch to project -> context everywhere as well?

Comment (williamkbentley) [reply src/frontend/app/AppState.ts]: Done

**Comment by calebmshafer** [comment]
@williamkbentley sorry for the delay in reviewing again, is this PR still valid?

**Comment by williamkbentley** [comment]
> @williamkbentley sorry for the delay in reviewing again, is this PR still valid?

Yeah, I think it should still be done. It brings the variable names more in line with our conventions.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**